com.bluejeans.BlueJeans.yaml

```diff
@@ -65,6 +65,7 @@ modules:
           - mv opt/bluejeans/* .
           - rm -r opt
           - chmod 755 bluejeans
+          - ARCH_TRIPLE=$(gcc --print-multiarch) && ln -s /usr/lib/${ARCH_TRIPLE}/libudev.so.1 /app/lib/libudev.so.0
```

Comment:
I may be wrong, but I don't think that using symlinks inside a Flatpak is a good idea.

Reply:
Why? This is common solution.

Reply:
Tons of examples: https://github.com/search?q=org%3Aflathub+ln+-s&type=Code

Reply:
Hmm, I thought that absolute paths in symlinks can be a problem for Flatpak. It seems that I was mistaken. @barthalion What is your opinion?

Reply:
I will merge the PR after this is approved by barthalion or other Flathub staff member.